webui/react/src/pages/TrialDetails/useTrialMetrics.ts

```diff
@@ -81,12 +56,50 @@ export const useTrialMetrics = (
 
       setData((prev) => {
         if (isEqual(prev, response)) return prev;
-        const trialData = response[0]?.metrics
-          .map((summMetric) => {
-            const key = metricToKey({ name: summMetric.name, type: summMetric.type });
-            return { [key]: summarizedMetricToSeries(summMetric) };
-          })
-          .reduce((a, b) => ({ ...a, ...b }), {});
+        const rawBatchValuesMap: Record<string, [number, number][]> = {};
```

Comment:
@nrajanee Could we move the creation of the `rawBatch`, `rawTime`, and `rawEpoch` values back into it's own function similar to `summarizedMetricToSeries` it would be cleaner to separate that logic from this `setData` method. Additionally, it would be helpful if we need to parse this response elsewhere.